pkgs/development/tools/click/default.nix

```
@@ -0,0 +1,83 @@
+{ lib
+, fetchFromGitLab
+, buildPythonApplication
+, autoreconfHook
+, debian
+, perl
+, vala
+, pkg-config
+, libgee
+, json-glib
+, properties-cpp
+, gobject-introspection
+, getopt
+, setuptools
+, pygobject3
+, wrapGAppsHook
+}:
+
+buildPythonApplication {
+  pname = "click";
+  version = "unstable-2023-02-22";
+
```

Nit: It's convention to set this up here, that way it's easier to tell why this isn't using regular Python build stuff.
```suggestion
  format = "other";

```